public/locales/pl/translation.json

```diff
@@ -180,5 +180,9 @@
   },
 
   "availabilityAlert": "Dostępność zostanie nadpisana wybranymi datami dla wszystkich ofert, które zaznaczysz.",
-  "noData": "Brak danych"
+  "noData": "Brak danych",
+
+  "imageCropperTitle": "Dostosuj zdjęcie",
+  "apply": "Zatwierdź",
+  "allowedPhotoFormats": "Dozwolone formaty zdjęć: jpg, jpeg, png, webp"
```

Comment:
Czy allowedPhotoFormats zostało też dodane do en?

Reply:
Chyba jest tylko w pl

Reply:
Faktycznie, wydawało mi się że dodawałem, możliwe że jak pullowałem zmiany z mastera to coś źle wyklikałem przy konfliktach

Reply:
Ok, to myślę, że można mergować